src/main/java/servlets/DataEntries.java

```diff
@@ -61,24 +57,26 @@ private String getJson() {
       return "{}";
     }
 
+    // Associate a runID with a list (for json) of the entry types that are found
+    // for it.
     HashMap<String, TreeSet<String>> dataMap = new HashMap<String, TreeSet<String>>();
```

Comment:
Does `TreeSet` get converted to a JSON array, or something more complicated?

Reply:
Yeah simple JSON array is the conversion GSON performs